code/modules/surgery/other.dm

```diff
@@ -245,8 +245,10 @@
 	possible_locs = list("head", "chest", "groin")
 
 /datum/surgery/remove_thrall/synth
+	name = "Debug Shadow Tumor"
```

Comment:
Doesn't debugging something mean it's kind of supposed to be there? I mean I can debug Paradise code (barely) but that doesn't mean it's getting a full removal like the tumor is going to. Unless I'm missing something wouldn't "deletion" be a better choice?